ant/src/controllers.lua

```diff
@@ -0,0 +1,52 @@
+local json = require("ant.src.json")
+local utils = require("ant.src.utils")
+
+Controllers = Controllers or {}
+
+local controllers = {}
+
+function controllers.setController(msg)
```

Comment:
same thing here, provide the actual values as parameters rather than passing the message. Parse the message in the Handler